Sources/Services/MockService.swift

```diff
@@ -74,10 +89,15 @@ let kTimeout: TimeInterval = 10
 	///   - consumer: Name of the API consumer (eg: "mobile-app")
 	///   - provider: Name of the API provider (eg: "auth-service")
 	///   - scheme: HTTP scheme
+	///   - port: The port number to run the MockServer on
 	///   - errorReporter: Injectable object to intercept errors
-	internal init(consumer: String, provider: String, scheme: TransferProtocol = .standard, errorReporter: ErrorReportable? = nil) {
+	internal init(consumer: String, provider: String, scheme: TransferProtocol = .standard, port: Int? = nil, errorReporter: ErrorReportable? = nil) {
 		pact = Pact(consumer: Pacticipant.consumer(consumer), provider: Pacticipant.provider(provider))
-		mockServer = MockServer()
+		if let port = port {
+			mockServer = MockServer(port: Int32(port))
+		} else {
+			mockServer = MockServer()
+		}
```

Comment:
`port` is optional on `MockServer.init` so you can remove the entire `if else` and pass directly the optional value.
```suggestion
		mockServer = MockServer(port: Int32(port))
```

Reply:
Can't cast a `nil` value to `Int32`. But good point and I'll move the check into `MockServer`.